data/campaigns/emp04.wmf/scripting/tribes/init.lua

```diff
@@ -61,24 +83,99 @@
 --   it will replace that tribe object.
 -- * You will also need to add a ``register.lua`` file that lists the names of all
 --   objects that you are adding or replacing.
--- * You can also add ``helptexts.lua`` and ``.png`` files to your subdirectories.
+-- * You can also add ``.png`` files to your subdirectories.
+-- * Adding helptexts is optional, but will units without a helptext will print a warning on the console.
 --
--- See :ref:`defining_tribe_units` for further details on these files.
+-- See :ref:`defining_tribe_units` for further details on these files, and :ref:`lua_tribes_tribes_helptexts` on the helptext format.
+
+-- For formatting time strings
+include "tribes/scripting/help/global_helptexts.lua"
+
+local buildingctxt = "empire_building"
```

Comment:
these variables have been dropped in the tribes units.lua files in the  commit fixing gettetxt errors. Are they safe here?